### PR TITLE
Add backup cadence variable to the appliance

### PR DESCRIPTION
**Pull request by Samirat**
In order to check if backups are failing or are not being taken on the appliance, I want to get the variable from here to indicate how often they _should_ be getting taken.

### CODE REVIEW

File: share/github-backup-utils/ghe-backup-mssql [hunk 1]
```diff
@@ -180,6 +180,9 @@ if [ -n "$backup_command" ]; then
   ghe-ssh "$GHE_HOSTNAME" -- "$backup_command" || failures="$failures mssql"
   bm_end "$(basename $0)"
 
+  # Configure the backup cadence on the appliance, which is used for diagnostics
+  ghe-ssh "$GHE_HOSTNAME" "ghe-config mssql.backup.cadence $GHE_MSSQL_BACKUP_CADENCE"
```
Comment (Samirat) [inline share/github-backup-utils/ghe-backup-mssql:184]: Have to change the script to handle this format.

Comment (boxofyellow) [reply share/github-backup-utils/ghe-backup-mssql:184]: So we consider making our live easier by change this to record the info we need.  The hours/days that back should happen.

Comment (Samirat) [reply share/github-backup-utils/ghe-backup-mssql:184]: I'd rather have it be the same everywhere and parse it when we use it.

Comment (boxofyellow) [reply share/github-backup-utils/ghe-backup-mssql:184]: Works for me.

**Review by boxofyellow — APPROVED** [review]

**Pull request merged**